TalismanSociety/signet-web
# [FEAT] - Vault description

# Description

Depends on: https://github.com/TalismanSociety/signet-backend/pull/18

## ⛑️ **What was done:**

- Vault Description can be added/edited on Settings.
- Vault Description will be displayed on Vault Overview

## Type of change


### 🧪 **How to test:**
1. Create a new Vault
2. Go to Settings
3. Add Multisig description

Expected result:
- Description should be updated on Settings
- Description should be displayed on Overview

### 🗒️ **Notes:**


### 🖼️ **Screenshots:**


https://github.com/TalismanSociety/signet-web/assets/47801291/458ead36-abad-41b9-8e7d-388e07ef179d
